satpy/readers/geocat.py

```diff
@@ -56,7 +56,30 @@
 
 
 class GEOCATFileHandler(NetCDF4FileHandler):
-    """GEOCAT netCDF4 file handler."""
+    """GEOCAT netCDF4 file handler.
+
+    **Loading data with decode_times=True**
+
+    By default, this reader will use xarray_kwargs={"engine": "netcdf4", "decode_times": False}.
+    to match behavior of xarray when the geocat reader was first written.  To use different options
+    use reader_kwargs when loading the Scene:
+
+        scene = satpy.Scene(filenames,
+                            reader='geocat',
+                            reader_kwargs={'xarray_kwargs': {'decode_times': True}})
+    """
+
+    def __init__(self, filename, filename_info, filetype_info,
+                 **kwargs):
+        """Open and perform initial investigation of NetCDF file."""
+        kwargs.setdefault('xarray_kwargs', {}).setdefault(
+            'decode_times', False)
+        kwargs.setdefault('xarray_kwargs', {}).setdefault(
+            'engine', "netcdf4")
+
+        super(GEOCATFileHandler, self).__init__(
+            filename, filename_info, filetype_info,
+            xarray_kwargs=kwargs["xarray_kwargs"])
```

Comment:
I think this should just be `**kwargs` in case the user is passing other kwargs down to the utility file handler.

Reply:
I think I understand where you want kwargs, in the  `__init__` correct?  The user would still send the xarray specific kwargs with the key "xarray_kwargs" so the documentation should reflect that.

Reply:
Correct. This call to `__init__` should use kwargs to pass everything to the base class. That doesn't change what the user uses.